.pre-commit-config.yaml

```
@@ -23,7 +23,7 @@ repos:
 -   repo: local
     hooks: 
```

```suggestion
    hooks:
```
There are also trailing whitespaces on line 17 and no newline at end of file.